### PR TITLE
Updated documentation 

### PR DESCRIPTION
**Pull request by carmenj**
This PR updates the documentation with the latest changes made to SDK and SDK UI, related to store and forward and wrapper refactoring.

### CODE REVIEW

File: README Developer Guide.md [hunk 1]
```diff
@@ -11,9 +11,25 @@ The enhanced sdk was developed to ease in the integration of the ClearentIDTechS
...
+    public func initialize(with config: ClearentWrapperConfiguration)
+```
+
+Because it's a static parameter, the configuration can be accessed like this:
+
+```
+   ClearentWrapper.configuration
+```
+
+**ClearentWrapperConfiguration** contains the following parameters: baseURL, apiKey, publicKey, enableEnhancedMessaging, enableOfflineMode, readerInfoReceived.
```
Comment (ovidiuvladpop) [inline README Developer Guide.md:26]: Is the "readerInfoReceived" parameter needed here as it was removed from the config's init?

File: README.md [hunk 2]
```diff
@@ -206,32 +196,36 @@ class ViewController: UIViewController {
     
     func initSDK() {
         
-        // Update the SDk with needed info to work properly
-        ClearentUIManager.shared.updateWith(baseURL: Constants.Api.baseURL, apiKey: Constants.Api.apiKey, publicKey: Constants.Api.publicKey, enableEnhancedMessaging: false)
-              
+        // Setup the SDK with needed config to work properly
+        let uiManagerConfiguration = ClearentUIManagerConfiguration(baseURL: "https....", apiKey: "api key...", publicKey: "public key", readerInfoReceived: nil)
```
Comment (ovidiuvladpop) [inline README.md:200]: Same thing here

File: README Developer Guide.md (hunk 1 above)
Comment (ovidiuvladpop) [reply README Developer Guide.md:26]: I think here is just specified as being a parameter of that class. Please correct me if I'm wrong.

File: README.md [hunk 3]
```diff
@@ -63,11 +63,11 @@ The SDK supports current version of iOS and two previous versions. Curently 13,
 
 ## How to Integrate
 
-In order to integrate the **SDK UI** you will need the **API URL**, **API KEY** and the **PUBLIC KEY**. 
-Use ClearentUIManager class to update the SDK with this information like this. 
+In order to integrate the **SDK UI** you will need to create a configuration object and pass it ClearentUIManager like in the following example: 
 
 ```
-ClearentUIManager.shared.updateWith(baseURL: baseURL, apiKey: apiKey, publicKey: publicKey)
+  let uiManagerConfig = ClearentUIManagerConfiguration(baseURL: baseURL, apiKey: apiKey, publicKey: publickKey, readerInfoReceived: nil, tipAmounts: [1, 2, 3], signatureEnabled: true)
```
Comment (ovidiuvladpop) [inline README.md:69]: Same thing here

File: README Developer Guide.md (hunk 1 above)
Comment (carmenj) [reply README Developer Guide.md:26]: indeed, it's needed because here is just specified as being a parameter of that class.

**Review by ovidiuvladpop — APPROVED** [review]

**Pull request merged**